ui/lib/core/addon/components/certificate-card.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import Component from '@glimmer/component';
+
+/**
+ * @module CertificateCard
+ * [Description]
+ *
+ * @example
+ * ```js
+ *  <CertificateCard @certificateValue={{value}} @isPem={{true}} />
+ * ```
+ * @param {string} certificateValue - the value to be displayed
+ * @param {boolean} isPem - optional argument for if the certificateValue is required to be PEM format
+ */
+
+export default class CertificateCardComponent extends Component {
```

Comment:
I have seen some components say something like `CertificateCardComponent` and others just say `CertificateCard` here. Is there a preferred syntax for this?
```suggestion
export default class CertificateCard extends Component {
```

Reply:
I prefer to be more verbose with class names in the event that there might be a service or something with the same name as a component but that's more personal preference. Either one would be fine with me.